test/target.spec.js

```diff
@@ -45,6 +45,15 @@ module.exports.addTests = function({testRunner, expect, puppeteer, browserWithEx
       await browserWithExtension.close();
       expect(backgroundPageTarget).toBeTruthy();
     });
+    it('target.page() should return a background_page', async({browser}) => {
+      const browserWithExtension = await puppeteer.launch(browserWithExtensionOptions);
+      const targets = await browserWithExtension.targets();
+      const backgroundPageTarget = targets.find(target => target.type() === 'background_page');
+      const page = await backgroundPageTarget.page();
+      expect(page).not.toBeNull();
+      await page.close();
```

Comment:
Can you please try evaluating something inside the page? It should be working, but I'd like to be sure.

Also: it's a little crazy to do `page.close` - iirc background page lifetime is controlled automatically. Let's drop the line?

Reply:
done. 10x!

Reply:
@aslushnikov - after rebasing master I have the [following test](https://github.com/GoogleChrome/puppeteer/blob/master/test/puppeteer.spec.js#L273) failing. The reason is that we now have a background page as well as the default `[about:blank]`, which I just added. 

Is this an indicator that we just broke compatibility here?